### PR TITLE
Fix `/serials` ralated code

### PR DESCRIPTION
**Pull request by okkez**

### CODE REVIEW

**Comment by mtsmfm** [comment]
We need some tests can detect this mistakes.

File: test/factories/serial.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    sequence(:slug) {|n| "serial#{n}" }
+    description { "#{title} description\n" }
+
+    trait :whatever do
+      site
+    end
+
+    trait :with_posts do
+      after :create do |serial, _evaluator|
+        serial.posts << build(:post, :whatever, site: serial.site)
```
Comment (tricknotes) [inline test/factories/serial.rb:9]: `:whatever` maybe unnecessary.

Comment (tricknotes) [inline test/factories/serial.rb:9]: Ouch. [`Post#category`](https://github.com/bm-sms/daimon-news/blob/master/test/factories/post.rb#L10) is needed.

File: test/factories/serial.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FactoryGirl.define do
+  factory :serial do
+    sequence(:title) {|n| "Serial #{n}" }
+    sequence(:slug) {|n| "serial#{n}" }
+    description { "#{title} description\n" }
+
+    trait :whatever do
```
Comment (tricknotes) [inline test/factories/serial.rb]: This seems be unused.

File: test/factories/serial.rb (hunk 1 above)
Comment (okkez) [inline test/factories/serial.rb:9]: I know `Post#category_id` is mandatory.
`:post`'s `:whatever` trait creates category automatically.
So I did not add `category` here.

File: test/factories/serial.rb (hunk 2 above)
Comment (okkez) [inline test/factories/serial.rb]: Removed

**Comment by okkez** [comment]
I added some commits for all comments.

File: test/integration/serial_test.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+class SerialTest < ActionDispatch::IntegrationTest
+  setup do
+    site = create(:site)
+    3.times do
+      create(:serial, :with_posts, site: site)
```
Comment (mtsmfm) [inline test/integration/serial_test.rb]: `create_list(:serial, 3)`

File: test/integration/serial_test.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    end
+
+    switch_domain(site.fqdn)
+  end
+
+  test "/serials" do
+    visit "/serials"
+
+    serial_titles = find_all "ul li p.serial-summary__title"
```
Comment (mtsmfm) [inline test/integration/serial_test.rb]: We prefer () for method call

Comment (mtsmfm) [inline test/integration/serial_test.rb]: Ah, but we don't prefer () for DSL and `find` is Capybara::DSL...

Comment (mtsmfm) [inline test/integration/serial_test.rb]: We need more discussion about our coding style, but I don't care about that on this PR.

**Comment by mtsmfm** [comment]
It works great :star: :shipit:

**Pull request merged**